.github/workflows/run_test.yaml

```diff
@@ -0,0 +1,39 @@
+name: Pytest
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
```

Comment:
Are we just testing for 3.11? Maybe add other versions like 3.10 or eeven 3.9.

Reply:
Related to [this comment](https://github.com/kalininalab/SMTB2024/pull/31#discussion_r1739983218). I'd test all Python versions that are actively developed, such as 3.12. But if we fix versions in the requirements, I don't see why we should test multiple versions. The installed packages will have the same versions across different tested Python versions.